Makefile

```diff
@@ -308,7 +306,35 @@ foss-checks: vendor
 # Unit Tests
 ###############################################################################
 ## Run the unit tests.
-ut: run-k8s-controller build $(BIN)/host-local
+ut:
+	make ut-etcd
+	make ut-kdd
+
+ut-etcd: run-k8s-controller build $(BIN)/host-local
```

Comment:
ut-kdd and ut-etcd both have the same dependencies.

It would be nice if we didn't need to restart the k8s components twice when running `make ut`.

Can we format this in a way to avoid that?

maybe something like this:

- `ut`: starts API, etc, and then runs UTs for all datastores.
- `ut-datastore`: runs UTs for a particular datastore, requires DATASTORE_TYPE argument, assumes everything is set up.
- `ut-kdd` starts API, etc, and then runs UTs for KDD by calling `ut-datastore`
- `ut-etcd` starts API, etc, and then runs UTs for etcd by calling `ut-datastore`

Reply:
Resolved in this commit: https://github.com/projectcalico/cni-plugin/pull/698/commits/67ece493d341032c9b39c0ee13824a3e680d1e39